packages/block-editor/src/components/responsive-block-control/style.scss

```diff
@@ -43,5 +31,14 @@
 }
 
 .block-editor-responsive-block-control .components-base-control__help {
```

Comment:
This doesn't feel right; if we need these styles here we should be applying them with whatever classname we're currently using to hide stuff from vision. Or we should be using `<VisuallyHidden>` if possible. Probably something to tackle in a follow-up PR though 😅. I don't think ResponsiveBlockControl is being used anywhere atm.